### PR TITLE
Daedalus fixes for master

### PR DESCRIPTION
**Pull request by deepfire**
This is https://github.com/input-output-hk/daedalus/pull/443 re-assembled, so it's mergeable into Daedalus `master`.

This PR, obviously won't function with `staging` without the corresponding addition of `daedalus-fixes-for-staging`.

To recap the summary:

0. some unused files were dropped (Darko)
1. a fix for OS X (Domen)
2. fix DEVOPS-290 -- we need proper NSIS in the build environment: https://issues.serokell.io/issue/DEVOPS-290
3. update Daedalus launch flags
4. since the `dns` library on Windows is broken, provide a topology file with _relay_ IP addresses hard-coded -- while this technically binds the installer to `staging`, the new Cardano can't be used on Windows otherwise anyway.
5. adapt `build-certiicates-win64.bat` to the changes that happened since it was introduced -- most importantly, make it use the `%APPDATA%` folder for intermediate/generated output.
6. fix the launch script constructor ending with a `^` -- this was causing a Haskell exception in `cardano-launcher`
7. the Daedalus log directory no longer exists prior to `build-certificates-win64.bat` execution, so it has to be created
8. the Electron's app is fixed to look for `tls/ca/ca.cert` in the proper location.

### CODE REVIEW

**Comment by domenkozar** [comment]
This is stil missing macOS fixes for TLS

**Comment by deepfire** [comment]
@domenkozar, addressed your suggestion.

File: electron/main.development.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ app.on('ready', async () => {
 
   try {
 
-    const ca = readCA(path.join(__dirname, '../tls/ca.crt'));
+    const ca = readCA(path.join(__dirname, '../../../tls/ca/ca.crt'));
```
Comment (darko-mijic) [inline electron/main.development.js]: This change is breaking TLS configuration in development. With this configuration TLS configuration file needs to be outside of the project directory. Something to discuss tomorrow @domenkozar and @deepfire.

Comment (akegalj) [reply electron/main.development.js]: Maybe we should put it in AppData as suggested by @domenkozar , where we have DB and logs

**Pull request merged**